project/SparkBuild.scala

```diff
@@ -33,13 +33,17 @@ object BuildCommons {
 
   private val buildLocation = file(".").getAbsoluteFile.getParentFile
 
-  val allProjects@Seq(bagel, catalyst, core, graphx, hive, hiveThriftServer, mllib, repl,
```

Comment:
`allProjects` is split into `nonStreamingProjects` and `streamingProjects` because Scala only support Tuple22 but here we have 23 projects, so `unapply` doesn't work here for a 23 items `Seq`.